tests/ballerina-spec-conformance-tests/S05_structured_values/tests/xml.bal

```diff
@@ -112,3 +112,27 @@ function testXmlSingleton() {
     xml x4 = x3.select("book");
     test:assertTrue(x4.isSingleton(), msg = "expected xml to be a singleton");
 }
+
+// An XML value is iterable as a sequence of its items, where each character item is
+// represented by a string with a single code point and other items are represented by a
+// singleton XML value.
+@test:Config {}
+function testXmlCharacterIteration() {
```

Comment:
Shall we merge this with https://github.com/ballerina-platform/ballerina-lang/pull/13893/files#diff-58a30b0864f7a00405d8634082c78661R73. Or at least move after `testXmlIteration`? We can remove the spec extract since it is already included.

Reply:
Done